### DISCUSSION TITLE
Power on password

### DISCUSSION THREAD

**Discussion by tuctuc22** [egzumer/uv-k5-firmware-custom#164] [General]
how enable power on password?

Thank u very much ..

**Comment by ErikS-web** [comment]
Not standard... see our [Online manual - section Differences from QS stock firmwarel](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Differences-from-QS-stock-firmware) And at last line how2doityourself

And for what reason ... than I suggest to use [Hiden Menu item 61](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#hidden-menu) and set it to **disables TX on all frequencies**
